### ISSUE TITLE
No default_validators

### ISSUE THREAD

**Issue by iraycd** [escaped/django-video-encoding#1]
In admin video upload is not because you have taken ImageField instead of FieldFile

**Comment by escaped** [comment]
Good catch. Haven't used my app with the django admin yet, which is why i didn't get that issue. Unfortunatly i am quite busy right now and have no time fixing that. So Pull Requests are welcome!
